en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/cc-rate-limiting.md

```diff
@@ -0,0 +1,51 @@
+# Rate Limiting
+
+Rate limiting allows users to limit the number of incoming requests to Choreo Connect. Choreo Connect connects
+with API Manager's Traffic Manager component in order to publish and receive Rate Limiting data, which is referred to as Global Rate Limiting.
+
+You can use the [distributed rate limiting]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/rate-limiting/distributed-throttling) option to globally rate limit requests.
+
+Choreo Connect supports Rate Limiting at the following levels:
+
+- API level
+- Resource level
+- Subscription level
+- Application level throttling
+
+## Distributed rate limiting
+
+In a deployment with multiple Choreo Connect instances, Rate Limiting becomes a challenge with node local Rate Limiting as the Rate Limiting decision is made based on the local counter within each node. If we proceed with the node local Rate Limiting in such
```

Comment:
```suggestion
In a deployment with multiple Choreo Connect instances, Rate Limiting becomes a challenge with node local Rate Limiting since the Rate Limiting decision is made based on the local counter within each node. If we proceed with the node local Rate Limiting in such
```